### PR TITLE
Change deprecated Layout to V1

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

-  Change deprecated Page Layout to v1
- Change width of LeftPanel
- Explorer, Glossary,Settings converted to v1
- fix the  vertical scroll issue in LeftPanel

#
### Type of change :

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css [hunk 1]
```diff
@@ -268,6 +268,10 @@
     @apply tw-border-b tw-border-separator;
   }
 
+  .activeCategory {
+    @apply tw-bg-primary-lite tw-border-l-2 tw-border-primary;
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css:272]: can we use custom css/less instead of using tailwind?

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment

File: openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css]: updated!

**Review by chirag-madlani — COMMENTED** [review]
@Ashish8689 please run cypress test locally and fix the missing steps

**Comment by ShaileshParmar11** [comment]
Can you please merge the main?

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**